tsconfig.json

```diff
@@ -6,20 +6,16 @@
     "module": "commonjs",
     "pretty": true,
     "noEmitOnError": true,
-    "experimentalDecorators": true,
-    "sourceMap": true,
```

Comment:
Why no source maps? I think they're still relevant on the server for debugging with the devtools, or for tooling like Sentry to pull in to show you the actual source that generated an error.

Reply:
Do source maps even work if no src folder is shipped? Regardelss, personally, I never ship sourcemaps since there is no need for the final user to debug library code.

Reply:
The source maps get written alongside the typescript compiler's output so yes they should still be included in the npm package. And I kind of feel like there isn't a downside to shipping them -- what happens if someone suspects there is a bug in this library, I feel like they would debug it wouldn't they? Especially if it's a pre-release library that has a higher likelihood of bugs, wouldn't we want to provide the best debugging experience possible?

Reply:
In my experience having a user that is going to debug a lib in node_modules is really rare. More often than not when something brakes they just open an issue, without even providing a repro repo. 

Usually, libraries don't ship maps because they are tested and hopefully maps can't provide any useful info. Maps, of course, are mainly used for application code. 

If it is important for you, we can re-add them, though.